go.mod

```diff
@@ -26,11 +27,13 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
-	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.14.0
-	github.com/tendermint/tendermint v0.33.7
+	github.com/tendermint/go-amino v0.15.0
```

Comment:
Please revert this

Reply:
So should we use `go-amino v0.15.0`  or `go-amino v0.16.0`? I'm getting confused :confused: 

@ethanfrey suggested to keep `v0.15.0` [here](https://github.com/cosmos/cosmos-sdk/pull/7531#discussion_r505245592)

Reply:
Ethan suggested to `downgrade go-amino back to the 0.15.x series`. Launchpad uses 0.15.1, thus I'd advise to just revert the change

Reply:
Ok, reverted back to 0.15.1